docs/decisions/0010-dotnet-project-structure.md

```diff
@@ -250,9 +250,11 @@ Below are some different examples of Assembly and root namespace naming that are
 ```text
 dotnet/
 ├── samples/
-│   ├── ApplicationInsightsExample/
-│   ├── KernelSyntaxExamples/
-│   └── NCalcSkills/
+│   └── Concepts/
+│   └── Demos/
+│   └── GettingStarted/
+│   └── GettingStartedWithAgents/
+│   └── LearnResources/
```

Comment:
We do not modify previous ADRs to keep history of changes/decisions. But you can mention that this ADR is superseded by a [new one](https://github.com/microsoft/semantic-kernel/blob/main/docs/decisions/0042-samples-restructure.md), like in this example:
https://github.com/microsoft/semantic-kernel/blob/246b84357ff6e0415fe5cb7fd97b622fb9bc9032/docs/decisions/0015-completion-service-selection.md?plain=1#L3

Reply:
I reflected your suggestion on [`ea88931` (#6403)](https://github.com/microsoft/semantic-kernel/pull/6403/commits/ea889317c0a08eaeae72ead9a66ef2816aef0a4e) . Thanks :)